### PR TITLE
Update M2K notifications

### PR DESCRIPTION
**Pull request by gabriel-farache**
Update M2K notifications:

- Add name of the branch and repo URL when sending successful notification
- Replace markdown with `actions` for move2kube Q&A waiting

Solves: 

- [FLPATH-890](https://issues.redhat.com/browse/FLPATH-890)
- [FLPATH-879](https://issues.redhat.com/browse/FLPATH-879)

### CODE REVIEW

File: move2kube/m2k.sw.yml [hunk 1]
```diff
@@ -93,9 +93,12 @@ states:
           refName: createNotification
           arguments:
             title: " \"Transformation \" + .transformationId + \" waiting for Q&A\" "
-            message: '"Please go to your <a href=\"" + $SECRET.move2kube_url + "/workspaces/"+ .workspaceId + "/projects/" + .projectId + "/outputs/"+ .transformId + "\">Move2Kube instance</a> and answers to questions in order to continue the Move2Kube workflow"'
+            message: '"Please go to your move2kube instance and answers to questions in order to continue the Move2Kube workflow"'
```
Comment (masayag) [inline move2kube/m2k.sw.yml:96]: s/answers to/answer the ?

File: move2kube/m2k.sw.yml [hunk 2]
```diff
@@ -177,7 +180,7 @@ states:
              refName: createNotification
              arguments:
                title: '"Move2Kube workflow " + $WORKFLOW.instanceId + " success"'
-               message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace " + .workspaceId + " and project " + .projectId + " was successful"'
-               origin: "Move2Kube Worflow"
+               message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace " + .workspaceId + " and project " + .projectId + " was successful; the output is available in the branch " + .targetBranch + " of your git repository "+ .repositoryURL"'
```
Comment (masayag) [inline move2kube/m2k.sw.yml:183]: is there an option to construct a link directly to the branch?

File: move2kube/specs/notifications.yaml [hunk 3]
```diff
@@ -238,6 +231,24 @@ components:
           type: string
         message:
           type: string
+        actions:
```
Comment (masayag) [inline move2kube/specs/notifications.yaml:234]: this will have to wait to one of two: PR to the generator or Pr to the notification plugin, right?

Comment (gabriel-farache) [reply move2kube/specs/notifications.yaml:234]: I see it on the main of janus idp (https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/src/openapi.yaml) is it not already merged? There are so many projects to propagate a change in a given project, I am totally lost as to where it should be merged to be available to us (the showcase I guess?)

File: move2kube/m2k.sw.yml (hunk 2 above)
Comment (gabriel-farache) [reply move2kube/m2k.sw.yml:183]: unfortunately, no; the URL is not the same if you use github/bitbucket/XXX so there is no way of providing an agnostic URL
And I am not keen at having if/else

**Comment by masayag** [comment]
> /hold for @mareklibra to public notification plugin as dynamic plugin

@gabriel-farache this can be unhold now.

**Comment by gabriel-farache** [comment]
\unhold

**Comment by gabriel-farache** [comment]
https://github.com/parodos-dev/orchestrator-helm-chart/pull/45 is including the notification plugin as a dynamic one in the charts

**Review by masayag — APPROVED** [review]

**Pull request merged**